P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -1732,11 +1732,11 @@ distinguish editorial changes from those visible in the source text.
 </p>
 <p>The <gi>ellipsis</gi> element provides a means to encode the deliberate, marked omission of content in a source document. 
   Within <gi>ellipsis</gi>, <gi>metamark</gi> describes the denotation of omitted material in the source, and encoders may optionally provide 
-  a <gi>note</gi> describing the omission and <gi>supplied</gi> to indicate the content omitted at this point. Here is an example of such an indicated omission from a manuscript:
+  a <gi>note</gi> describing the omission and <gi>supplied</gi> to indicate the content omitted at this point. We provide an example of such an indicated omission from a manuscript.
   <figure><graphic url="Images/cod-10991.jpg" mimeType="image/jpeg"/>
```

Comment:
I've just noticed that the source of the manuscript page is not given in the caption (resp. in the <head>).

Reply:
Ah yes! I've just added the source to the `<head>`!

Reply:
(Also revised the `<head>` so it's a little easier to read now, I think.)